docs/central-users.rst

```diff
@@ -19,67 +19,94 @@ Central features Role-based User permissioning. In the current release of Centra
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Edit & Review                    | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Comment                          | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View & Download                  | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List All                         | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Add Properties                   | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update Settings                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entities**                                                                                 |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create from Submission           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create Directly                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View in Linked Forms             | x             | x               | x              | x              |
```

Comment:
Here, I list "view" before "update", but for submissions, it's "view" that comes last. I thought "view" should come earlier here, because the two types of view actions contrast in a similar way as the two types of create actions directly above.